### ISSUE TITLE
document.getElementById(...).create is not a function

### ISSUE THREAD

**Issue by kwge** [Polymer/test-fixture#16]
Keep getting document.getElementById(...).create is not a function
    <unknown> at       context.fixture at /components/test-fixture/test-fixture-mocha.js:32:0
when running with web-component-tester on line 
var btn = fixture('paper-button');

**Comment by kwge** [comment]
Include the wrong header.  It is working now

**Comment by inkubux** [comment]
Can you please elaborate on which header you included wrongly ?

I have the same issue and can't find why 

Thanks a lot

**Comment by tpluscode** [comment]
@kwge I see the same problem on Travis. What was your solution?

**Comment by sohailsomani** [comment]
Try including the `webcomponents` script in your HTML file.

**Comment by tpluscode** [comment]
How did I miss this reply..

@sohailsomani you mean include in the test HTML files? [I do](https://github.com/tpluscode/ld-navigation/blob/master/test/ld-navigation-history-tests.html#L5)

**Comment by sviridovserg** [comment]
@tpluscode  check where you create fixture. Should be in setup/beforeEach, otherwise you'll see this error "document.getElementById(...).create is not defined". I'm pretty sure it is connected with how different browsers load files included via `<link rel="import" href="...">`

**Comment by tpluscode** [comment]
I most certainly call `fixture` from `beforeEach` every time

**Comment by drewswaycool** [comment]
@tpluscode have you found a resolution to the issue? I am having the same problem.

**Comment by tpluscode** [comment]
no, sorry

**Comment by zliebersbach** [comment]
Same issue. Any tips?

**Comment by justinadesriuteRE** [comment]
after changing the order of imports (<link rel="import" href="my-custom-element"> now placed before .../webcomponents-lite.js and .../web-component-tester/browser.js in the <head>) I stopped receiving the mentioned error but then started getting "Polymer is not defined". 
Anyone had the same problem?

**Comment by leventku** [comment]
This issue happened to me **only** for firefox (version 54). The reason turned out to be the hyphen casing I used in the fixture's id field `<test-fixture id="my-component-fixture">`.

Changing to camel casing solved the issue: `<test-fixture id="MyComponentFixture">`

**Comment by tylergraf** [comment]
I've just been wrapping my `describe` in `document.addEventListener('WebComponentsReady', function(){`. I'd rather not have to do that. Any ideas?

**Comment by thetrevoradams** [comment]
I also had this issue, and found by updating `web-component-tester` to the latest version (currently `6.4.0`) fixed the problem.

**Comment by ghost** [comment]
In one case moving to 6.4.3 worked for me a la @ta1188.

In another case camel casing my test fixture worked a la @leventku 

What the hell?

**Comment by jlk4p** [comment]
Ugh! I was having the same issue this morning. Turns out that some old way of coding a component test was the problem after I reviewed fixture names (which were fine) and updated my WCT. It looks like Firefox needs any variable/constant defined for the fixture inside the test where it is referenced. Once I moved that inside the test function, FF testing worked without any problems and produced the same results as Chrome.
